alacritty.yml

```diff
@@ -592,8 +592,7 @@
 # - Vi mode exclusive actions:
 #
 #   - Open
-#       Perform the action of the first matching hint, with `mouse.enabled`,
-#       under the cursor.
+#       Perform the action of the first matching hint under the vi mode cursor with `mouse.enabled` set to `true`.
```

Comment:
Could you wrap this for me? IIRC it should all be wrapped at `80`.

Reply:
I just noticed that myself, and already fixed it.